docs/README.md

```diff
@@ -1,114 +1,232 @@
 # ScalarDB Benchmarks
 
-This repository contains benchmark programs for ScalarDB.
+This tutorial describes how to set up and run benchmarking tools for ScalarDB.
 
 ## Available workloads
 
 - TPC-C
-- YCSB (Workload A, C, and F)
-- Multi-storage YCSB (Workload C and F)
-  - A YCSB-variant benchmark using a multi-storage environment of ScalarDB.
-  - Workers in multi-storage YCSB execute the same number of read/write operations in two namespaces (`ycsb_primary` and `ycsb_secondary`).
+- Yahoo! Cloud Serving Benchmark (YCSB) (Workloads A, C, and F)
+- Multi-storage YCSB (Workloads C and F)
+  - This YCSB variant is for a multi-storage environment that uses ScalarDB.
+  - Workers in a multi-storage YCSB execute the same number of read and write operations in two namespaces: `ycsb_primary` and `ycsb_secondary`.
 
 ## Prerequisites
 
-- Java (OpenJDK 8 or higher)
+- One of the following Java Development Kits (JDKs):
+  - [Oracle JDK](https://www.oracle.com/java/technologies/downloads/) LTS version 8
+  - [OpenJDK](https://openjdk.org/install/) LTS version 8
 - Gradle
-- Kelpie
+- [Kelpie](https://github.com/scalar-labs/kelpie)
+  - Kelpie is a framework for performing end-to-end testing, such as system benchmarking and verification. Get the latest version from [Kelpie Releases](https://github.com/scalar-labs/kelpie), and unzip the archive file.
 
-The benchmark uses Kelpie, which is a simple yet general framework for performing end-to-end testing such as system benchmarking and verification. Get the latest version of Kelpie from [here](https://github.com/scalar-labs/kelpie) and unzip the archive.
+{% capture notice--info %}
+**Note**
 
-## Usage
+Currently, only JDK 8 can be used when running the benchmarking tools.
+{% endcapture %}
 
-### Set up an environment
+<div class="notice--info">{{ notice--info | markdownify }}</div>
 
-This benchmark requires the followings:
-- A client to execute this benchmark
-- A target database (See [here](https://github.com/scalar-labs/scalardb/blob/master/docs/scalardb-supported-databases.md) for supported databases)
+## Set up your environment
 
-### Build
+The benchmarking tools require the following:
+
+- A client to run the benchmarking tools
+- A target database
+  - For a list of databases that ScalarDB supports, see [Supported Databases](https://github.com/scalar-labs/scalardb/blob/master/docs/scalardb-supported-databases.md).
```

Comment:
I think the following bullet points would be better under the `## Prerequisites` section. 

Would that be OK? Or do we need to add more information to this section?